plugins/mcreator-core/blockly/js/mcreator_blocks.js

```diff
@@ -782,5 +785,24 @@ Blockly.defineBlocksWithJsonArray([
         ],
         "output": "Direction",
         "colour": "20"
+    },
+    {
+        'type': 'controls_flow_statements',
+        'message0': '%1',
+        'args0': [
+            {
+                'type': 'field_dropdown',
+                'name': 'FLOW',
+                'options': [
+                    ['%{BKY_CONTROLS_FLOW_STATEMENTS_OPERATOR_BREAK}', 'BREAK'],
+                    ['%{BKY_CONTROLS_FLOW_STATEMENTS_OPERATOR_CONTINUE}', 'CONTINUE'],
+                ],
+            },
+        ],
+        'previousStatement': null,
+        'style': 'loop_blocks',
+        'helpUrl': '%{BKY_CONTROLS_FLOW_STATEMENTS_HELPURL}',
+        'suppressPrefixSuffix': true,
+        'extensions': ['controls_flow_tooltip', 'controls_flow_in_loop_check_exclude_wait'],

```

Comment:
Would it maybe be better to `addChangeListener()` to `workspace` for events of type `Blockly.Events.BLOCK_CREATE` that would register all needed custom extensions to predefined blocks using `Blockly.Extensions.apply('<extensionId>', this, <isMutator>)`?

Reply:
You can't register apply extensions to already registered blocks. Check Blockly docs

Reply:
I would not do something authors explicitly said not to do…

Reply:
Resolved then.